Taarafo.Core/Brokers/Storages/IStorageBroker.PostReports.cs

```diff
@@ -4,12 +4,14 @@
 // ---------------------------------------------------------------
 
 using System.Linq;
+using System.Threading.Tasks;
 using Taarafo.Core.Models.PostReports;
 
 namespace Taarafo.Core.Brokers.Storages
 {
     public partial interface IStrorageBroker
     {
+        ValueTask<PostReport> InsertPostReportAsync(PostReport postReport);
```

Comment:
This PR is for Delete Group Memebership.  This is out of scope for this change please exclude this change and do this as a seperate PR.